test/e2e/pkg/infra/digitalocean/digitalocean.go

```diff
@@ -0,0 +1,91 @@
+package digitalocean
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	e2e "github.com/tendermint/tendermint/test/e2e/pkg"
+	"github.com/tendermint/tendermint/test/e2e/pkg/exec"
+	"github.com/tendermint/tendermint/test/e2e/pkg/infra"
+)
+
+var _ infra.Provider = (*Provider)(nil)
+
+// Provider implements a DigitalOcean-backed infrastructure provider.
```

Comment:
wooo, so we will support running e2es in a cloud provider? crazy

Reply:
they have that in v0.38, so we have it here. we could do this in theory, but imo if we want that we should just use the app.

so I'd be down for deleting these